try:
    a == 1
except Exception, e:
    print "here"
    print e

